phpstan.neon

```diff
@@ -22,3 +22,10 @@ parameters:
         - '#Method CommerceWeavers\\SyliusTpayPlugin\\Form\\Type\\TpayCardType::__construct\(\) has parameter \$cardTypeDataTransformer with generic interface Symfony\\Component\\Form\\DataTransformerInterface but does not specify its types: TValue, TTransformedValue#'
         - '#Parameter \#2 \$listener of method .+::addEventListener\(\) expects callable\(\): mixed, array.+ given\.#'
         - '#Constructor of class CommerceWeavers\\SyliusTpayPlugin\\Test\\Payum\\Cypher\\FakeCypher has an unused parameter \$cypherKey\.#'
+        - '#Parameter \$stateMachine of method CommerceWeavers\\SyliusTpayPlugin\\Payment\\Checker\\PaymentCancellationPossibilityChecker::__construct\(\) has invalid type Sylius\\Abstraction\\StateMachine\\StateMachineInterface\.#'
+        - '#Property CommerceWeavers\\SyliusTpayPlugin\\Payment\\Checker\\PaymentCancellationPossibilityChecker::\$stateMachine has unknown class Sylius\\Abstraction\\StateMachine\\StateMachineInterface as its type\.#'
+        - '#Call to method can\(\) on an unknown class Sylius\\Abstraction\\StateMachine\\StateMachineInterface\.#'
+        - '#Parameter \$stateMachine of method CommerceWeavers\\SyliusTpayPlugin\\Payment\\Canceller\\PaymentCanceller::__construct\(\) has invalid type Sylius\\Abstraction\\StateMachine\\StateMachineInterface\.#'
+        - '#Property CommerceWeavers\\SyliusTpayPlugin\\Payment\\Canceller\\PaymentCanceller::\$stateMachine has unknown class Sylius\\Abstraction\\StateMachine\\StateMachineInterface as its type\.#'
+        - '#Caught class Sylius\\Abstraction\\StateMachine\\Exception\\StateMachineExecutionException not found\.#'
+        - '#Call to method apply\(\) on an unknown class Sylius\\Abstraction\\StateMachine\\StateMachineInterface\.#'
```

Comment:
Why? Do you know what's the problem behind that?

Reply:
It does not exist on Sylius `1.12`, so I silence it; it's strictly related to supporting two Sylius versions at the same time

Reply:
Ok, I see. However, we could take advantage of that it's a regex:
```suggestion
        - '#has invalid type Sylius\\Abstraction\\StateMachine\\StateMachineInterface#'
        - 'has unknown class Sylius\\Abstraction\\StateMachine\\StateMachineInterface#'
        - 'an unknown class Sylius\\Abstraction\\StateMachine\\StateMachineInterface#'
        - '#Caught class Sylius\\Abstraction\\StateMachine\\Exception\\StateMachineExecutionException not found\.#'
        - '#Call to method apply\(\) on an unknown class Sylius\\Abstraction\\StateMachine\\StateMachineInterface\.#'
```